config/initializers/omniauth.rb

```diff
@@ -12,7 +12,8 @@
            },
            name: 'CCID'
 
-  provider :developer unless Rails.env.production?
+  provider :developer unless (Rails.env.production? ||
+                              ENV['ENABLE_TMP_LOGINS'].present?)
```

Comment:
This conditional logic is still not quite right.

Previously it was `unless (Rails.env.production? && !ENV['ENABLE_TMP_LOGINS'])` which means it will only be enabled if it was both not in production AND the env var was set. Which isn't correct as it should be OR.

Now its ` unless (Rails.env.production? || ENV['ENABLE_TMP_LOGINS'].present?)` which means its only enabled if its not in production OR ENV VAR is not present. Which is not true as we want the setting of the ENV VAR in production to allow this to be turned on?

Also `unless` gets very confusing with you have complicated conditionals (with `&&`, `||`) so maybe it be easier if you switched it to an `if`?

Basically you want  `if !Rails.env.production? ||  ENV['ENABLE_TMP_LOGINS'].present?`